lib/src/contextual-menu/ContextualMenu.test.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import DxcContextualMenu from "./ContextualMenu.tsx";
+
+const items = [{ label: "Item 1" }, { label: "Item 2" }, { label: "Item 3" }, { label: "Item 4" }];
+
+const sections = [
+  {
+    title: "Team repositories",
+    items: [{ label: "Approved locations" }, { label: "Approved locations" }, { label: "Approved locations" }],
+  },
+  {
+    items: [{ label: "Approved locations" }, { label: "Approved locations" }, { label: "Approved locations" }],
+  },
+];
```

Comment:
Slots could be added to some of the items (badges and SVGs).

Reply:
Do they add anything relevant to the unit tests? They are checked on the storybooks.